CONTRIBUTING.md

```diff
@@ -1,9 +1,11 @@
+# Contributing to Playful Programming Blog
+
 As a community, we welcome anyone to write and submit posts to the Playful Programming blog! In this document, we'll go over how to create a new post in UU and submit it as a pull request.
 
-> [!NOTE]
-> For a general tutorial on contributing to a project with GitHub, you may want to go through [the First Contributions guide](https://github.com/firstcontributions/first-contributions) before continuing through this page.
+**🚀 IMPORTANT: If you are new to contributing via GitHub, consider going through the [First Contributions guide](https://github.com/firstcontributions/first-contributions) before proceeding.**
```

Comment:
I'd rather we keep the `[!NOTE]` formatting here, since it's a standard of GitHub and makes highlighting a little more obvious